src/test/java/seedu/address/testutil/PersonUtil.java

```diff
@@ -82,17 +83,25 @@ public static String getSellerDetails(Seller seller) {
         sb.append(PREFIX_NAME + seller.getName().fullName + " ");
         sb.append(PREFIX_PHONE + seller.getPhone().value + " ");
         sb.append(PREFIX_EMAIL + seller.getEmail().value + " ");
-        sb.append(PREFIX_HOUSING_TYPE + seller.getHousingType() + " ");
+        sb.append(PREFIX_HOUSING_TYPE + seller.getHousingType().value + " ");
         // Append house details
         for (House house : seller.getHouses()) {
             sb.append(PREFIX_STREET + house.getStreet().value + " ");
-            if (house instanceof NonLanded) {
-                NonLanded nonLanded = (NonLanded) house;
-                if (nonLanded.getBlock() != null) {
-                    sb.append(PREFIX_BLOCK).append(nonLanded.getBlock().value).append(" ");
+            if (house instanceof Hdb) {
+                Hdb hdb = (Hdb) house;
+                if (hdb.getBlock() != null) {
+                    sb.append(PREFIX_BLOCK).append(hdb.getBlock().value).append(" ");
                 }
-                if (nonLanded.getLevel() != null) {
-                    sb.append(PREFIX_LEVEL).append(nonLanded.getLevel().value).append(" ");
+                if (hdb.getLevel() != null) {
+                    sb.append(PREFIX_LEVEL).append(hdb.getLevel().value).append(" ");
+                }
+            } else if (house instanceof Condominium) {
+                Condominium condominium = (Condominium) house;
+                if (condominium.getBlock() != null) {
+                    sb.append(PREFIX_BLOCK).append(condominium.getBlock().value).append(" ");
+                }
+                if (condominium.getLevel() != null) {
+                    sb.append(PREFIX_LEVEL).append(condominium.getLevel().value).append(" ");
```

Comment:
I think you're missing Landed here

Reply:
Landed does not have Block or Level and is the default case